ui/StatusQ/src/StatusQ/Core/StatusListView.qml

```diff
@@ -1,5 +1,5 @@
-import QtQuick 2.14
-import QtQuick.Controls 2.14
+import QtQuick 2.15
+import QtQuick.Controls 2.15
```

Comment:
We are not there yet :)

Reply:
I know. It's a draft for when we get there :D. There is still some work to be done here after we migrate to 5.15